packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -90,12 +113,17 @@ export default class VTable {
                     }
                 });
             });
+            const formatInfo = getTableFormatInfo(this.table);

```

Comment:
It is possible the retrieved format info is invalid. At least we should make sure it has all values for all properties.